### DISCUSSION TITLE
FermionSpinSum over only final state spinor chain

### DISCUSSION THREAD

**Discussion by mjb659** [FeynCalc/feyncalc#288] [Q&A]
Hello, I am trying to compute the amplitude for a 2 fermion to 2 fermion process. I want to perform FermionSpinSum over the outgoing fermion spins. However, I do not want to sum over the initial fermion spins, but instead be able to set their specific spin state. It looks like FermionSpinSum performs the sum over all closed spinor chains in a squared amplitude, but is there a way to achieve this sum over only the final state spin chain, and specify the initial spinors by hand?
Thank you for the help!

**Comment by vsht** [comment]
You can rename the spinors that you don't want to sum over to something else, so FermionSpinSum won't touch them.
